### PR TITLE
Small UI fixes

### PR DESCRIPTION
**Pull request by scottbommarito**
https://github.com/NuGet/NuGetGallery/issues/5820
https://github.com/NuGet/NuGetGallery/issues/5741
https://github.com/NuGet/NuGetGallery/issues/5746
https://github.com/NuGet/NuGetGallery/issues/5770
https://github.com/NuGet/NuGetGallery/issues/5807
https://github.com/NuGet/NuGetGallery/issues/5817

image

### CODE REVIEW

File: src/NuGetGallery/Services/MessageService.cs [hunk 1]
```diff
@@ -638,29 +638,17 @@ private void SendSupportMessage(User user, string body, string subject)
 
         public void SendPackageUploadedNotice(Package package, string packageUrl, string packageSupportUrl, string emailSettingsUrl)
         {
-            string subject = "[{0}] Package uploaded - {1} {2}";
-            string body = @"The package [{1} {2}]({3}) was just uploaded to {0}. If this was not intended, please [contact support]({4}).
+            string subject = $"[{Config.GalleryOwner.DisplayName}] Package uploaded - {package.PackageRegistration.Id} {package.Version}";
+            string body = $@"The package [{package.PackageRegistration.Id} {package.Version}]({packageUrl}) was just uploaded to {Config.GalleryOwner.DisplayName} by {package.User.Username}. If this was not intended, please [contact support]({packageSupportUrl}).
```
Comment (loic-sharma) [inline src/NuGetGallery/Services/MessageService.cs]: >just [](start = 112, length = 4)

Nit: "was just uploaded" -> "was uploaded" / "was recently uploaded" ?

File: src/NuGetGallery/Views/Packages/ConfirmOwner.cshtml [hunk 2]
```diff
@@ -71,7 +71,7 @@
                 @* Note that we're not confirming that a request even exists for this user, it's just a string compare against the name in the URL *@
                 @ViewHelpers.AlertDanger(
                     @<text>
-                        This request link is for the user '@Model.Username'. If you wish to accept it,
+                        This request link is for the user '@Model.Username'. If you want to accept it,
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ConfirmOwner.cshtml]: >want [](start = 84, length = 4)

Nit: "want" seems a little strong. What do you think of "would like"?

File: src/NuGetGallery.Core/Services/CoreMessageService.cs [hunk 3]
```diff
@@ -26,27 +26,15 @@ public CoreMessageService(IMailSender mailSender, ICoreMessageServiceConfigurati
 
         public void SendPackageAddedNotice(Package package, string packageUrl, string packageSupportUrl, string emailSettingsUrl)
         {
-            string subject = "[{0}] Package published - {1} {2}";
-            string body = @"The package [{1} {2}]({3}) was just published on {0}. If this was not intended, please [contact support]({4}).
+            string subject = $"[{CoreConfiguration.GalleryOwner.DisplayName}] Package published - {package.PackageRegistration.Id} {package.Version}";
+            string body = $@"The package [{package.PackageRegistration.Id} {package.Version}]({packageUrl}) was just published on {CoreConfiguration.GalleryOwner.DisplayName} by {package.User.Username}. If this was not intended, please [contact support]({packageSupportUrl}).
```
Comment (loic-sharma) [inline src/NuGetGallery.Core/Services/CoreMessageService.cs]: >just [](start = 112, length = 4)

Same feedback as before here.

File: src/NuGetGallery/Views/Packages/ConfirmOwner.cshtml (hunk 2 above)
Comment (scottbommarito) [reply src/NuGetGallery/Views/Packages/ConfirmOwner.cshtml]: replaced every "want" with "would like" lol

File: src/NuGetGallery/Services/MessageService.cs (hunk 1 above)
Comment (scottbommarito) [reply src/NuGetGallery/Services/MessageService.cs]: done

File: src/NuGetGallery.Core/Services/CoreMessageService.cs (hunk 3 above)
Comment (scottbommarito) [reply src/NuGetGallery.Core/Services/CoreMessageService.cs]: done

**Pull request merged**